src/Front/DesugarStrings.hs

```diff
@@ -19,8 +22,37 @@ desugarStrings (Program decls) = return $ Program $ desugarTop <$> decls
 
   desugarDef (DefFn v bs t e) = DefFn v bs t $ desugarExpr e
   desugarDef (DefPat b e    ) = DefPat b $ desugarExpr e
+  
```

Comment:
Replace lines 23 and 24 with
```
  desugarDef (DefFn v bs t e) = DefFn v bs t $ everywhere (mkT desugarExpr) e
  desugarDef (DefPat b e    ) = DefPat b $ everywhere (mkT desugarExpr) e
```
You will likely need to import Data.Generics to make `everywhere` and `mkT` work.